### ISSUE TITLE
The dashboard UI is too old.

### ISSUE THREAD

**Issue by ganshanshan** [spring-cloud/spring-cloud-dataflow-server-kubernetes#230]
Hi , contributors

@mstine @royclarkson @jstrachan @rmorgan @jvalkeal 

I have installed the spring-cloud-dataflow-server-kubernetes with the lastest version(1.3.0.M1) and create the docker images with command
`./mvnw package docker:build -pl :spring-cloud-dataflow-server-kubernetes`

and login the system with 

> http://<ip>:80/dashboard

but the dashboard UI is older than 'Local Server' deploy type. here is the stream tab shotscren.

![image](https://user-images.githubusercontent.com/19347709/30264961-86927d1e-968f-11e7-926c-8362ae5cba00.png)

So i want to confirm with you that 
1.the version of dashboard UI is different from 'Local Server'.
2.On which version do you plan to update the new dashboard UI version.

Thanks.

**Comment by ilayaperumalg** [comment]
Hi,

The dashboard UI uses `1.3.0.M1` version of the Spring Cloud data flow UI which is being migrated to Angular 4. You can find more information on this [here](https://spring.io/blog/2017/08/07/spring-cloud-data-flow-1-3-0-m1-released). The upcoming version `1.3.0.M2` will have more updates to the UI.

**Comment by trisberg** [comment]
Thanks for trying this out. The 1.3.0.M1 release is an early milestone release with the only real change being the UI which is still in development. We included the most recent milestone of the UI when we made the release, but as you noticed you would have to use the Data Flow shell to create streams and tasks. I would recommend staying with 1.2.2.RELEASE for now, until the 1.3.0.M2 release, when we will have a more complete new UI implementation. Now, we also have a Helm chart for the 1.2.2.RELEASE to make it easier to install, see - https://spring.io/blog/2017/08/31/simple-installation-of-data-flow-for-kubernetes-with-helm
